flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/abilities/source/WatermarkPushDownSpec.java

```diff
@@ -52,21 +54,27 @@
 public final class WatermarkPushDownSpec extends SourceAbilitySpecBase {
     public static final String FIELD_NAME_WATERMARK_EXPR = "watermarkExpr";
     public static final String FIELD_NAME_IDLE_TIMEOUT_MILLIS = "idleTimeoutMillis";
+    public static final String FIELD_NAME_WATERMARK_PARAMS = "watermarkParams";
 
     @JsonProperty(FIELD_NAME_WATERMARK_EXPR)
     private final RexNode watermarkExpr;
 
     @JsonProperty(FIELD_NAME_IDLE_TIMEOUT_MILLIS)
     private final long idleTimeoutMillis;
```

Comment:
I think this field should be renamed as `globalIdleTimeoutMillis`

Reply:
renamed